web/src/theme/_main.scss

```diff
@@ -1,5 +1,5 @@
-@import '../../node_modules/angular-tree-component/dist/angular-tree-component.css';
-@import 'bootstrap-3-backward-compatibility';
+@import '../../node_modules/ngx-bootstrap/datepicker/bs-datepicker';
```

Comment:
We replaced the bootstrap-datepicker with the datepicker from angular material, so I'm not sure whether we need this import. However, removing all files belonging to the unused bootstrap-datepicker can be done in separate ticket as well.